lib/ansiblelint/utils.py

```diff
@@ -52,6 +53,26 @@
 
 PLAYBOOK_DIR = os.environ.get('ANSIBLE_PLAYBOOK_DIR', None)
 
+INVALID_CONFIG_RC = 2
+ANSIBLE_FAILURE_RC = 3
+
+VERBOSITY_MAP = {
+    0: logging.NOTSET,
+    1: logging.INFO,
+    2: logging.DEBUG
+}
+
+logger = logging.getLogger()
```

Comment:
Not sure if I want only one global logger to be used. Why didn't you go in the direction of per-module loggers?

Reply:
Yep, better even if ansiblelint is a cli tool.

Reply:
Could you please elaborate on this? Not sure I can parse this.